docs/src/pages/tutorial/run-in-docker.md

```diff
@@ -3,17 +3,24 @@ id: run-monika-in-docker
 title: Run Monika in Docker
 ---
 
-Monika is available as a docker image. Paste a copy of your configuration file `monika.yml` into the current directory.
-From the same directory, you can run:
+Monika is available as a docker image. You can find the image in the docker hub as `hyperjump/monika`, or pull the image with the following command:
+
+```bash
+docker pull hyperjump/monika
+```
+
+Once you've pulled the latest image, pass it your monika yaml configuration to the container. From the same directory, you can run:
 
 ```bash
 # Run Monika in foreground
 docker run --name monika -v ${PWD}/monika.yml:/config/monika.yml --detach hyperjump/monika:latest
```

Comment:
this command with `--detach` flag is conflicting with the `foreground` in the comment above mas

Reply:
oops thanks. copas. fixed